src/components/layout/Navbar/index.tsx

```diff
@@ -30,17 +51,27 @@ const Navbar = ({ user }: NavbarProps) => {
   return (
     <header className={styles.header}>
       <div className={styles.content}>
-        <Link href={config.homeRoute} className={styles.icon}>
+        {/* Mobile Navbar Toggle */}
+        <button
+          type="button"
+          className={`${styles.toggleIcon} ${!mobile && styles.hidden}`}
```

Comment:
Same thing as the comment below on data-attributes but we could describe this as something like
`
<button type="button" className={styles.toggleIcon} data-state={mobile ? "visible" : "hidden"}>
`
```
.toggleIcon[data-state="visible"] {
}
```

Also, I would also flip from calling it hidden to visible if I was keeping the current behavior to avoid a extra negation on the `mobile` boolean and make the code easy to comprehend for anyone reading.